src/tests/test_dynamic_api/test_views_wfs.py

```diff
@@ -92,3 +92,18 @@ def test_wfs_field_auth(
             "geometry": None,
             "straatnaam": None,
         }
+
+    def test_wfs_feature_name(
+        self, api_client, filled_router, afval_dataset, afval_adresloopafstand
+    ):
+        """Prove that if feature name contains non-letters like underscore,
+        it can be useds find the correct table name and data
+        """
+        print("******", afval_adresloopafstand.__dict__)
+        wfs_url = (
+            "/v1/wfs/afvalwegingen/"
+            "?SERVICE=WFS&VERSION=2.0.0&REQUEST=GetFeature&TYPENAMES=app:adres_loopafstand"
+            "&OUTPUTFORMAT=application/gml+xml"
+        )
+        response = api_client.get(wfs_url)
+        assert response.status_code == 200
```

Comment:
I think checking status is not enough anymore (as error in getting details results in same 200, but with error in xml).
Would be good to check if model is loaded and at least empty list is given back. Ideally create instance of AdresLoopafstand and check if it is returned as WFS.

Reply:
Check! Added extra tests based on presence of output (content)